### DISCUSSION TITLE
How to count a string/ a particular channel in mdf file?

### DISCUSSION THREAD

**Discussion by markusmat97** [danielhrisca/asammdf#701] [Q&A]
My mdf file contain a channel RUT_vel_0,RUT_vel_1......to RUT_vel_49.In some mdf files, this channel count is 40 and it always different for some files.Now i want to get the number of the string "RUT_vel_".How to count the number of particular channel or the string in mdf?

**Comment by danielhrisca** [comment]
You can start by using the mdf.channels_db dictionary
